patches/react-nft-gallery+0.6.2.patch

```diff
@@ -0,0 +1,422 @@
+diff --git a/node_modules/react-nft-gallery/dist/react-nft-gallery.esm.js b/node_modules/react-nft-gallery/dist/react-nft-gallery.esm.js
```

Comment:
this is a pretty big diff to have to do against the transpiled code. Understand the urgency, so makes sense to do it this way temporarily.

Reply:
Right, see my comment above.